sklearn/linear_model/ridge.py

```diff
@@ -960,9 +986,7 @@ def fit(self, X, y, sample_weight=None):
         error = scorer is None
 
         for i, alpha in enumerate(self.alphas):
-            weighted_alpha = (sample_weight * alpha
-                              if sample_weight is not None
-                              else alpha)
+            weighted_alpha = alpha
```

Comment:
If we are no longer reweighting alpha (as it seems to be the case if I read this right), shouldn't we avoid the renaming, and keep the name "alpha"?